tests/modelci/test_unit_test.py

```diff
@@ -2,29 +2,24 @@
 from tests.helpers import (
     EVAL_FUNCTION_COMPARISON,
     EVAL_FUNCTION_THRESHOLD,
-    TEST_EVAL_FUNCTION_ID,
     TEST_SLICE_NAME,
     get_uuid,
 )
 from tests.test_dataset import make_dataset_items
 
 
-def test_unit_test_metric_creation_from_class(unit_test):
+def test_unit_test_metric_creation(CLIENT, unit_test):
     # create some dataset_items for the unit test to reference
-    unit_test_metric = unit_test.add_metric(
-        eval_function_id=TEST_EVAL_FUNCTION_ID,
-        threshold=EVAL_FUNCTION_THRESHOLD,
-        threshold_comparison=EVAL_FUNCTION_COMPARISON,
-    )
+    iou = CLIENT.modelci.eval_functions.iou
+    unit_test_metric = unit_test.add_criteria(iou() > EVAL_FUNCTION_THRESHOLD)
     assert unit_test_metric.unit_test_id == unit_test.id
     assert unit_test_metric.eval_function_id
     assert unit_test_metric.threshold == EVAL_FUNCTION_THRESHOLD
     assert unit_test_metric.threshold_comparison == EVAL_FUNCTION_COMPARISON
 
-    metrics = unit_test.get_metrics()
+    metrics = unit_test.get_criteria()
```

Comment:
nit: can we also rename the variable to `criteria`?

Reply:
As long as it's a function it should be `get_...`, if it's `criteria` only it should become a property.